packages/store-ui/src/molecules/Bullets/stories/Bullets.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Story } from '@storybook/react'
+import React from 'react'
+
+import type { BulletsProps } from '../Bullets'
+import Component from '../Bullets'
+import mdx from './Bullets.mdx'
+
+const BulletsTemplate: Story<BulletsProps> = (args) => <Component {...args} />
```

Comment:
I think we can destructure these args into { totalQuantity, activeBullet, onClick, testId, ariaLabelGenerator}, to make it clear on the Story tab what we are doing. I'm not sure, what do you think?

<img width="432" alt="Screen Shot 2021-06-24 at 4 30 51 PM" src="https://user-images.githubusercontent.com/42946693/123322440-b354cb00-d50a-11eb-8bb4-754b369e4baa.png">

Reply:
Awesome! I agree!